azure_arc_k8s_jumpstart/alibaba/terraform/providers.tf

```diff
@@ -0,0 +1,13 @@
+#
+# Providers Configuration
+#
+
+terraform {
+  required_version = "~> 0.12"
+  required_providers {
+    alicloud  = "~> 1.119"
+    random = "~> 3.1"
+  }
+}
+
+provider "alicloud" {}
```

Comment:
```suggestion
provider "alicloud" {}

```

Reply:
as I am using environment variables I followed this notation https://registry.terraform.io/providers/aliyun/alicloud/latest/docs#environment-variables with an empty provider spec